package.json

```diff
@@ -17,10 +17,10 @@
   },
   "scripts": {
     "clean": "rimraf .awcache .cache-loader build node_modules transpiled",
-    "build": "webpack --bail",
+    "build": "webpack --bail --mode=production",
```

Comment:
Is this `mode` necessary since it’s in all the config files? Same with `start-dev-server` below...

Reply:
It's necessary here, yeah.  This command line flag gets passed into the `configure` function (in `webpack.common.js`) as `webpackOptions.mode`.  

I suppose it's not necessarily required for `start-dev-server` (because `development` is the default) but I like it being explicit personally.